Python/Extract_Contents_of_Zip/script.py

```diff
@@ -0,0 +1,15 @@
+from zipfile import ZipFile
+
+def main():
+    # Enter the name of zip file
+    zipFile = input("Enter zip file:")
+    # Enter the path of the file to be extracted
+    extractFile = input("Enter path of the file:")
```

Comment:
you can take any one input from user. Either file name or path. are both really required?

Reply:
I have made the changes.